components/devfile-page/Header.tsx

```diff
@@ -4,7 +4,7 @@ import HeaderTags from './HeaderTags';
 import { Brand, Card, CardBody, Text, TextContent, TextVariants } from '@patternfly/react-core';
 
 /**
- * props for devPage metadata header
+ * props for devfile page metadata header
```

Comment:
can you change the const name to `header`

Reply:
resolved